recipes/liblsl/all/conanfile.py

```diff
@@ -25,18 +26,6 @@ class LiblslConan(ConanFile):
         "fPIC": True,
     }
 
-    exports_sources = "CMakeLists.txt"
-    generators = "cmake", "cmake_find_package_multi"
-    _cmake = None
-
-    @property
-    def _source_subfolder(self):
-        return "source_subfolder"
-
-    @property
-    def _build_subfolder(self):
-        return "build_subfolder"
-
     def requirements(self):
         self.requires("boost/1.78.0")
```

Comment:
```suggestion
        self.requires("boost/1.81.0")
```
Could you please use the latest Boost version available, so we will not have conflict errors in case boost being consumed by other packages too.